lang/cs/Org.Apache.REEF.Network/Naming/NameClient.cs

```diff
@@ -62,15 +64,21 @@ public sealed class NameClient : INameClient
         /// </summary>
         /// <param name="remoteAddress">The ip address of the NameServer</param>
         /// <param name="remotePort">The port of the NameServer</param>
+        /// <param name="connectionRetries">Number of retries for connection to endpoint</param>
+        /// <param name="sleepTime">Sleep time in milliseconds between retries</param>
         [Inject]
         private NameClient(
             [Parameter(typeof(NamingConfigurationOptions.NameServerAddress))] string remoteAddress,
-            [Parameter(typeof(NamingConfigurationOptions.NameServerPort))] int remotePort)
+            [Parameter(typeof(NamingConfigurationOptions.NameServerPort))] int remotePort,
+            [Parameter(typeof(EndpointConnectionParameters.EndpointConnectionRetries))] int connectionRetries,
+            [Parameter(typeof(EndpointConnectionParameters.SleepTimeWaitingForConnection))] int sleepTime)
```

Comment:
Please use `TimeSpan` for parameters specifying duration. 
If there is *really* a genuine reason for using only primitive types then use descriptive variable names like `sleepTimeInMs`.